samples/XML/address_decode_v2.circ

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@turtlebasket Are there many `.circ` files that lack an XML header? If not, then these files should be classified correctly as XML on GitHub, as Linguist has a [dedicated strategy](https://github.com/github-linguist/linguist/blob/ff004eaeb54e868e3b1707f0584ceb3684b155f2/lib/linguist/strategy/xml.rb) for recognising files that begin with the line

```xml
<?xml version=
```

If many `.circ` files in-the-wild lack these headers and aren't receiving any highlighting, then this addition is still valid (and you can dismiss this feedback).